src/components/UserManagement/SetUpFinalDayButton.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Button } from 'reactstrap';
+import { SET_FINAL_DAY, CANCEL } from '../../languages/en/ui';
+import SetUpFinalDayPopUp from './SetUpFinalDayPopUp';
+import { updateUserFinalDayStatus } from 'actions/userManagement';
+import axios from 'axios';
+import { ENDPOINTS } from 'utils/URL';
+import { toast } from 'react-toastify';
+/**
+ * @param {*} props
+ * @param {Boolean} props.isBigBtn
+ * @param {*} props.userProfile.isSet
+ * @returns
+ */
+const SetUpFinalDayButton = (props) => {
+  const[isSet, setIsSet] = useState(false);  
+  const[finalDayDateOpen, setFinalDayDateOpen] = useState(false);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (props.userProfile?.endDate !== undefined)
+      setIsSet(true);
+  }, [setIsSet]);
+  
+  const onFinalDayClick = (user, status) => {
+    if(isSet){
+      updateUserFinalDayStatus(props.userProfile,"Active",undefined)(dispatch)
+      setIsSet(!isSet);
+      toast.success('Your Final Day has been set.');
+      window.location.reload();
+      
+    }else{
+      setFinalDayDateOpen(true);
+    }
+    
+  };
+  const setUpFinalDayPopupClose = () => {
+    setFinalDayDateOpen(false);
+  };
+    
+  const deactiveUser = (finalDayDate) => {
+    updateUserFinalDayStatus(props.userProfile, "Active", finalDayDate)(dispatch);
+    setIsSet(true);
+    setFinalDayDateOpen(false);
+    toast.success('Your Final Day has been deleted.');
+    window.location.reload();
+  }
+
+  return (
+    <React.Fragment>
+       <SetUpFinalDayPopUp
```

Comment:
line 52 need rm one indent to match line 56

Reply:
thx for the comment, i will watch out the coding style more carefully.